spa/.env

```
@@ -1,5 +1,5 @@
 # Polling for Windows
-VITE_USE_POLLING=false
+VITE_USE_POLLING=true
```

```suggestion
VITE_USE_POLLING=false
```
Mettre true dans un .env.local

---

Pitié désolé maître